test/e2e/cluster_extension_install_test.go

```diff
@@ -713,8 +713,8 @@ func TestClusterExtensionInstallReResolvesWhenNewCatalog(t *testing.T) {
 		assert.Contains(ct, cond.Message, "resolved to")
 		assert.Equal(ct,
 			&ocv1alpha1.ClusterExtensionResolutionStatus{Bundle: &ocv1alpha1.BundleMetadata{
-				Name:    "prometheus-operator.1.2.0",
-				Version: "1.2.0",
+				Name:    "prometheus-operator.2.0.0",
+				Version: "2.0.0",
```

Comment:
The original test seemed wrong. The catalog is updated to v2.0, but it remains as 1.2.0? That doesn't seem right. With the latest changes this now seems to be behaving as expected.